Bio/Blast/Applications.py

```diff
@@ -715,14 +715,14 @@ def __init__(self, cmd="tblastn", **kwargs):
                     "Minimum score for words to be added to the BLAST lookup table (float).",
                     equate=False),
             _Option(["-comp_based_stats", "comp_based_stats"],
-                    """Use composition-based statistics (string, default 2, i.e. True).
+                    r"""Use composition-based statistics (string, default 2, i.e. True).
 
                     0, F or f: no composition-based statistics
                     1: Composition-based statistics as in NAR 29:2994-3005, 2001
                     2, T or t, D or d : Composition-based score adjustment as in
-                       Bioinformatics 21:902-911, 2005, conditioned on sequence properties
+                    \   Bioinformatics 21:902-911, 2005, conditioned on sequence properties
                     3: Composition-based score adjustment as in Bioinformatics 21:902-911,
-                       2005, unconditionally
+                    \   2005, unconditionally
```

Comment:
Why this slash here and on the second bullet point? These docstrings should be valid RST.